src/pages/HistoryPage/getHistoryData.ts

```diff
@@ -111,6 +111,21 @@ export const getHistoryData = async (circleId: number, userId: number) => {
                         },
                       ],
                     },
+                    epochStatements: {
+                      histories: [
+                        {},
+                        {
+                          id: true,
+                          bio: true,
```

Comment:
looks like we're not using all these fields, right? can we remove all that arent sued.